### PR TITLE
C4A-644 Food name - CR Help guide link has no behavior when Help guide is already open.

### PR DESCRIPTION
**Pull request by alex-oommen**

### CODE REVIEW

File: src/components/partials/help-guide-lb-business.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ export default () => html`
     </section>
   </div>
 
-  <div class="side-padding vertical-padding">
+  <div id="foodWithExtraReq" class="side-padding vertical-padding">
```
Comment (duttonw) [inline src/components/partials/help-guide-lb-business.js:91]: is id="foodWithExtraReq"
all on the same page?
if so this is against dom spec which is id's must be unique.
please use a class instead and make id's unique.

File: src/components/help-guide.js [hunk 2]
```diff
@@ -256,6 +256,15 @@ export class HelpGuide {
    * @return {HTMLTemplate}
    */
   createTemplate(state) {
+    // ----- This code is used to scroll to "Food with extra requirements"
+    // Section in Help guide ----- //
+    const helpGuideButton = document.querySelector("button[data-accordion-item='help-guide']");
+    if (helpGuideButton) {
+      helpGuideButton.addEventListener('click', () => {
+        document.getElementById('foodWithExtraReq').scrollIntoView();
```
Comment (duttonw) [inline src/components/help-guide.js:264]: swap from ById to document.getElementsByClassName

Comment (alex-oommen) [reply src/components/help-guide.js]: Changed to getElementsByClassName().

File: src/components/partials/help-guide-lb-business.js (hunk 1 above)
Comment (alex-oommen) [reply src/components/partials/help-guide-lb-business.js]: Changed Id to class

**Review by duttonw — APPROVED** [review]

**Pull request merged**